centreon/packages/ui/jest.config.js

```diff
@@ -1,24 +1,24 @@
-const baseConfig = require('../js-config/jest');
+const baseConfig = require("../js-config/jest");
```

Comment:
any particular reason you changed the format ?

Reply:
Is it an auto-apply configuration from webstorm?

Reply:
nope, IDE that does not pick up the prettier/lint config well from the monorepo

Reply:
thought i fixed

Reply:
fixed